renderapi/image_pyramid.py

```diff
@@ -37,90 +40,130 @@ def _formatUrls(self):
             d.update({'maskUrl': self.maskUrl})
         return d
 
+    def __getitem__(self, key):
```

Comment:
should this also support assignment with a setitem?  The current implementation does not support 
```python
ip[0]['imageUrl'] = "file:///path/to/image"
```

Reply:
oh i don't see why not... might actually make it more backwards compatible.